hrjobcontract/CRM/Hrjobcontract/Dedupe/Merger.php

```diff
@@ -1488,7 +1732,7 @@ static function moveAllBelongings($mainId, $otherId, $migrationInfo) {
 
           case 'Select Country':
           case 'Select State/Province':
-            $submitted[$key] = CRM_Core_BAO_CustomField::getDisplayValue($value, $fid, $cFields);
+            $submitted[$key] = CRM_Core_BAO_CustomField::displayValue($value, $fid);
```

Comment:
I really don't like this change. getDisplayValue and displayValue have totally different meanings!
I know this is not something you changed, but I just wanted to comment anyway 😜